### DISCUSSION TITLE
New Source: Primetric

### DISCUSSION THREAD

**Discussion by andrzejdackiewicz** [airbytehq/airbyte#36090] [New Connector Request]
## Tell us about the new connector you’d like to have
I need to have a source-primetric connector.

## Describe the context around this new connector
* I am working on data injestion from a variety of systems used in the company into our data lake. I found out that Airbyte is missing one of the required sources. We need the data for analysis. Some internal processes like finding people for projects can be faster using the data.
* I will most likely need a 1/day batch, overwrite syncs
* Assignments, Employees, Projects are necessary for me

## Describe the alternative you are considering or using
Well I am using Airbyte existing source definitions and they work fine so that would be sad. I do not want to write any script that would do the work (outside of Airbyte). That would be a hustle to maintain.

## Are you willing to submit a PR? 
Yes! I would love to submit a PR with the code for my missing connector.

**Comment by andrzejdackiewicz** [comment]
I would like to work on this issue :smiley:

**Comment by marcosmarxm** [comment]
Awesome @andrzejdackiewicz I assigned the issue to you. Let me know if you need any assistance here.

**Comment by andrzejdackiewicz** [comment]
Hello! I added a MR to Airbyte. Please take a look when You have time :smiley: 
https://github.com/airbytehq/airbyte/pull/15880

**Comment by marcosmarxm** [comment]
Sure, thanks so much for the contribution!

[a comment by an automated account is not shown]
